### PR TITLE
Blog views

### PR DESCRIPTION
**Pull request by abdulhamiid**
Hi, :wave: 
 I have implemented the project requirements in the link below
### [Project Requiremts](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -10,21 +14,13 @@
...
-    <h3>Post 2</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
-  </div>
-  <div>
-    <h3>Post 3</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
+    <% @user.posts.each do |post| %>
+      <h3>
```
Comment (omar-muhamad) [inline app/views/users/show.html.erb]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `recent_posts` method in the `user` model to render only the last 3 recent posts.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -10,21 +14,13 @@
...
+      <%= link_to link_to post.title, user_post_path(@user.id, post.id) %>
+      </h3>
+      <p><%= post.text %></p>
+    <% end %>
   </div>
   <button>See all posts</button>
```
Comment (omar-muhamad) [inline app/views/users/show.html.erb]: - [ ] Clicking on `see all posts` suppose to opens user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png), So please fix that by using `link_to` or `button_to` method.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,29 +1,23 @@
...
+  <% @user.posts.each do |post| %>
+    <article>
+      <h2>
+      <%= link_to link_to post.title, user_post_path(@user.id, post.id) %>
+      </h2>
+      <p><%= post.text %></p>
+      <span>Comments: <%= post.comments_counter%>,</span>
+      <span>Likes: <%= post.likes_counter %></span>
+      <% post.comments.each do |comment| %>
+        <p>Username: <%= comment.text %></p>
```
Comment (omar-muhamad) [inline app/views/posts/index.html.erb]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use `five_recent_comments` method to render the 5 most recent comments.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @abdulhamiid,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/posts/index.html.erb (hunk 3 above)
Comment (abdulhamiid) [reply app/views/posts/index.html.erb]: Alright, I will make the changes

**Review by Davidosky007 — APPROVED** [review]
Hi @abdulhamiid 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Good looking PR description 👍 
* No linter errors 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abdulhamiid** [comment]
Thank you!

**Pull request merged**